prow/getting_started_deploy.md

```diff
@@ -16,9 +16,14 @@ You need a few things:
 1. The prow `tackle` utility. It is recommended to use it by running `bazel run //prow/cmd/tackle` from `test-infra` directory, alternatively you can install it by running `go get -u k8s.io/test-infra/prow/cmd/tackle` (in that case you would also need go installed and working).
 1. A [personal access token][1] for the GitHub bot user you want to act as prow. If you're setting up prow for personal repo (not organization) then bot user == you (your GitHub account).
 
-   - Token must have `public_repo` and `repo:status` scopes (and repo for private repos), more details [here][8].
-   - Prow uploads this token into a secret in your cluster
-   - Bot user must have write access to the repo
+  - Add the following scopes for the token (more details [here][8])
+    - Must have the `public_repo` and `repo:status` scopes
+    - Add the `repo` scope if you plan on handing private repos
+    - Add the `admin_org:hook` scope if you plan on handling a github org
+  - Ensure the bot user has the following permissions
+    - Write access to the repos you plan on handling
+    - Owner access to the orgs you plan on handling (it is possible to handle specific repos in an org without this)
```

Comment:
this is _only_ required if you run `peribolos` or `branch-protector` which are peripheral to Prow

Reply:
I found that `tackle` was unable to take advantage of the `admin_org:hook` scope I setup for the bot user's token until I made it an Owner

Reply:
The other tools you mention likely require more scopes, I'm guessing, to do things like org membership changes etc.  I haven't gotten there yet